test/integration/common_functions.sh

```diff
@@ -180,20 +180,12 @@ ghcr_update() { # $1: file
     yq e -i '.kubernetes.deployment.image.tag = env(TAG)' $1
     # add imagePullSecrets
     yq e -i '.kubernetes.deployment.image.imagePullSecrets[0].name = env(IMAGEPULLSECRET)' $1
-    # add cosign validator:
```

Comment:
This change completely changes the context of this function. It should at least be renamed. Probably, we can get rid of it, too

Reply:
Maybe even the whole stuff related to GHCR can be scrubbed from the testcases

Reply:
the ghcr image is still present, this much hasn't changed. only thing that has changed is that the image is no longer signed. thus the function is still valid.